test_scripts/Resumption/InteriorVehicleData/002_InteriorVD_resumption_one_moduleType_with_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case app was subscribed to one module with moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1, moduleId)
```

Comment:
please change all module_1 to moduleType_1

Reply:
@yang1070 It is has been done in all scripts of this PR